src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -201,8 +149,21 @@ class AddressValidationModal extends React.Component {
       addressValidationType,
       suggestedAddresses,
       addressFromUser,
+      validationKey,
+      addressValidationError,
+      // eslint-disable-next-line no-shadow
```

Comment:
I normally hate ignoring linter rules, but I'm actually in favor of this one for this particular use case. I actually wish I'd thought of it the many times I've run into this problem.

Reply:
Actually, I don't think this linter rule should be ignored, since ignoring it can cause confusion.